extensions/2.0/Khronos/KHR_materials_sheen/README.md

```diff
@@ -0,0 +1,226 @@
+# KHR\_materials\_sheen
+
+## Contributors
+
+* Sebastien Vandenberghe, Microsoft, [@sebavan](https://github.com/sebavan)
+* Alexis Vaginay, Microsoft, [@Popov72](https://github.com/Popov72)
+* Tobias Häußler, Dassault System, [@proog128](https://github.com/proog128)
+* Ed Mackey, Analytical Graphics, Inc.
+* Romain Guy, Google, [@romainguy](https://github.com/romainguy)
+* Mike Bond, Adobe, [@MiiBond](https://github.com/MiiBond)
+* Don McCurdy, Google, [@donmccurdy](https://twitter.com/donrmccurdy)
+* Gary Hsu, Microsoft, [@bghgary](https://twitter.com/bghgary)
+* Eliphas NUIT, Dassault System, [@EliphasNUIT](https://github.com/EliphasNUIT)
+* Aidin Abedi, Epic Games, [@aidinabedi](https://github.com/aidinabedi)
+
+Copyright (C) 2018-2020 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Exclusions
+* This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+* This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+This extension defines a sheen that can be layered on top of an existing glTF material definition. A sheen layer is a common technique used in Physically-Based Rendering to represent cloth and fabric materials, for example. See [Theory, Documentation and Implementations](#theory-documentation-and-implementations)
+
+## Extending Materials
+
+The PBR sheen materials are defined by adding the `KHR_materials_sheen` extension to any compatible glTF material (excluding those listed above). 
+For example, the following defines a material like velvet.
+
+```json
+{
+    "materials": [
+        {
+            "name": "velvet",
+            "extensions": {
+                "KHR_materials_sheen": {
+                    "sheenColorFactor": [0.9, 0.9, 0.9]
+                }
+            }
+        }
+    ]
+}
+```
+
+### Sheen
+
+All implementations should use the same calculations for the BRDF inputs. Implementations of the BRDF itself can vary based on device performance and resource constraints. See [appendix](/specification/2.0/README.md#appendix-b-brdf-implementation) for more details on the BRDF calculations.
```

Comment:
I think it would be beneficial to mark the sheen (BRDF) implementation as non-normative - this goes along with the glTF 2.0 BRDF implementation.
Implementers are free to choose some other formula for the BRDF but this is the one we suggest for realtime rasterizers and will be implemented in the Sample-Viewer.

Reply:
We are using the exact same phrasing than the other extensions which should be ok ??? or we might need to revisit the one on clear coat and so on as well. I do think it is actually what this means ???

Reply:
I understand the current phrasing as exactly what you're saying @rsahlin. The BRDF input computations based on the parameters need to comply to the spec, which is crucial for consistency. But the implementation itself is non-normative and may vary depending on the constraints. (this might also play a significant role in consistency, but is inevitable.)

Reply:
@sebavan In the referenced glTF documentation (Appendix B) linked by clearcoat it states:
> This section is non-normative.

This goes for the whole BRDF implementation - I think that all parameters added by extensions must follow that. 
So yes - I think we must update clearcoat to match.

For glTF to incease adoption as a  file transmission format I believe that it is very important to keep the mindset that we are defining a set of parameters.
Exactly how these parameters are used is up to implementations.
Mandating how the values are used is up to certification, whose purpose is to enable consistensy and predictability in viewers.

Reply:
So the parameter list is normative, and the suggested implementation with formulas are non-normative, I think that sounds correct.  This means the parameter list will need to appear in a different section from the non-normative implementation notes and formulas, which means a change to the wording in clearcoat.

Reply:
When writing extensions (#1718, #1719, #1825, #1726) I separated the list of parameters and their behavior, described with text and a set of images, from the implementation part. While the implementation is non-normative, the behavior and intent of parameters is not. Behavior and intent includes how factor and texture are combined, how the effect blends with other effects (layering, mixing, Fresnel, etc.), a description of the physical effect we want to simulate, and so on.

In other words:
* Information you need to create an MDL or OSL shader should be part of the normative section.
* Information you need to create a renderer for this MDL shader (i.e., implement the MDL/OSL nodes/functions in a renderer with functions like sample/eval/pdf/...) are part of the non-normative section.

This ensures that the material renders consistently everywhere (same parameters with same behavior, for example base color changes color of diffuse BRDF and microfacet BTDF), but still gives enough room for custom, renderer-specific optimizations (for example the choice of BSDF for global illumination vs. real-time, like Estevez-Kulla vs. Ashikhmin visibility term).

In the future I hope we can adopt a common representation for the normative part which is used in all extensions. Describing the behavior of paramters with text may lead to misunderstandings. I drafted this in #1717 and #1738. Please have a look at especially the rewrite of the clearcoat extension in #1717. I am planning to update this for all extensions as soon as they are merged.

Reply:
I opened #1836 to update clearcoat.

Reply:
Done as proposed on clear cloat in #1836